eth/eth.go

```diff
@@ -0,0 +1,73 @@
+// contains ethereum specific data structures and functions
+package eth
+
+import (
+	"errors"
+
+	"github.com/decred/dcrd/dcrec/secp256k1/v4"
+	"github.com/decred/dcrd/dcrec/secp256k1/v4/ecdsa"
+	"golang.org/x/crypto/sha3"
+)
+
+type (
+	Hash    [32]byte
+	Address [20]byte
```

Comment:
where is this type used?

Reply:
Not anywhere. I am just playing around with ideas on this one.